src/helpers/OTSubscriberHelper.js

```diff
@@ -53,30 +53,30 @@ const sanitizeResolution = (resolution) => {
   if (
     typeof resolution !== 'object' ||
     (resolution &&
-      resolution.width === void 0 && // TODO use typeof !== 'number'
-      resolution.height === void 0) ||
+      resolution.width === undefined && // TODO use typeof !== 'number'
```

Comment:
You no longer need to use `void 0` (and the linter complains about it). As of ES5 (which is 15 years old), you can safely use `undefined` (see [this write-up](https://dev.to/p42/you-dont-need-void-0-663)).